x/pairing/types/expected_keepers.go

```diff
@@ -99,7 +100,8 @@ type DowntimeKeeper interface {
 type DualstakingKeeper interface {
 	RewardProvidersAndDelegators(ctx sdk.Context, providerAddr sdk.AccAddress, chainID string, totalReward math.Int, senderModule string, calcOnly bool) (providerReward math.Int, err error)
 	Delegate(ctx sdk.Context, delegator, provider, chainID string, amount sdk.Coin) error
```

Comment:
let's remove delegate access now and make it private, should use Delegatefull now

Reply:
done